index.html

```diff
@@ -199,5 +191,40 @@ <h1 class="about-title">About me</h1>
                 </div>
         </div>
     </section>
+    <section class="contact">
+        <div class="cont">
+            <h2 class="contact-text">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
+            <form action="https://formspree.io/f/xvoyeaqj" method="post">
+                <ul class="forms">
+                    <li>
+                        <input type="text" name="name" class="input" id="name" placeholder="Full name">
+                        <label for="name" class="label">Full name</label>
+                    </li>
+                    <li>
+                        <input type="email" name="email" class="input" id="email" placeholder="Email">
+                        <label for="email" class="label">Email</label>
+                    </li>
+                    <li> 
+                        <textarea class="area" rows="5" name="message">Enter text here</textarea>
+                    </li>
+                    <li>
+                        <input type="submit" name="submit" value="Get in touch" class="submit">
+                    </li>
```

Comment:
- According to the project requirements, all input fields should have the `required` attribute to ensure your HTML form validation, please try to fix this 👍 👍 
- There are some little differences between your design and the Figma design you have implemented. Remember it is important that you try to replicate the Figma design as closely as possible to give confidence to recruiters in your ability to produce exact business requirements 🙏

Your project:
-- The Figma template does not have labels for the input fields, but if you must use it, for a better user experience, I suggest that your label for each input field should be above the input field, please fix this 🙏 
-- The background color for the contact section uses hex-code `fff`. Please fix this 👍 👍 
-- Please implement button interactions such as hover, active (when it's pressed), disabled, and enable (default state) styled as shown in the Figma template for your submit button, optionally, you can consider doing so for your other buttons.
**OPTIONAL** - I suggest you work on your footer icons so they look close to what is in the design template 👍 

Your project:
<img width="158" alt="gwrgwrg" src="https://user-images.githubusercontent.com/73607512/205876631-547accd6-39df-4b71-9ac2-5db555b69a7a.PNG">

Figma template:
<img width="146" alt="ethsethseth" src="https://user-images.githubusercontent.com/73607512/205876692-9885b9e0-5e7d-4a69-9c55-f64c8c87cc70.PNG">

Button interactions in Figma:
<img width="344" alt="whrhaeae" src="https://user-images.githubusercontent.com/73607512/205876732-dbfe989b-40e0-4dae-be80-d5abd2ff3df7.PNG">